addons/remove-search-bar-autocomplete/addon.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Remove autocomplete from search bar",
+  "description": "Stops web browsers from autocompleting suggestions in the navigation bar's search bar.",
+  "tags": ["community"],
+  "credits": [
+    {
+      "name": "jbthepig",
+      "link": "https://scratch.mit.edu/users/jbthepig"
+    }
+  ],
+  "dynamicEnable": true,
+  "userscripts": [
```

Comment:
Could you attempt adding dynamicDisable to this addon? You can use an [`addon.self.addEventListener` on "disabled" and "reenabled"](https://scratchaddons.com/docs/reference/addon-api/addon.self/#disabled) to achieve this. If not, I can take care of it for you :)